mne/viz/utils.py

```diff
@@ -1507,6 +1507,7 @@ def _plot_sensors(pos, colors, bads, ch_names, title, show_names, ax, show,
                          ch_names=ch_names, show_names=show_names)
         fig.canvas.mpl_connect('pick_event', picker)
 
+    ax.axis("off")  # remove border around figure
```

Comment:
I noticed though, that it does not show the coordinate axis names (the x, y and z). So for 3d plotter:
```
ax.xaxis.set_label_text('x')
ax.yaxis.set_label_text('y')
ax.zaxis.set_label_text('z')
```